src/main/io/osd.c

```diff
@@ -668,6 +671,21 @@ static bool osdDrawSingleElement(uint8_t item)
                 break;
             }
 
+#ifdef USE_ESC_SENSOR
+            // Show warning if we lose motor output
+            if (enabledWarnings & OSD_WARNING_ESC_FAIL && ARMING_FLAG(ARMED)) {
+                for (int i = 0; i < getMotorCount(); i = i + 1) {
```

Comment:
Why you don't use the habitual i++ form?
```C
 for (int i = 0; i < getMotorCount(); i++)
```

Reply:
Yes please, consistency is key.

Reply:
I saw it used elsewhere so just went with it - will change.